src/entries/popup/pages/swap/index.tsx

```diff
@@ -161,6 +161,58 @@ const SwapWarning = ({
   );
 };
 
+const MissingPriceExplanation = ({
+  assetToBuy,
+  assetToSell,
+}: {
+  assetToBuy: ParsedSearchAsset | null;
+  assetToSell: ParsedSearchAsset | null;
+}) => {
+  const missingPriceSymbols = [assetToBuy, assetToSell].reduce(
+    (symbols, asset) => {
+      if (asset?.price?.value === undefined && asset?.symbol) {
+        return [...symbols, asset?.symbol];
+      }
+      return symbols;
+    },
+    [] as string[],
+  );
+  if (!missingPriceSymbols?.length) return null;
```

Comment:
I don't think this can ever be undefined?
```suggestion
  if (!missingPriceSymbols.length) return null;
```